app/models/health_care_application.rb

```diff
@@ -271,8 +271,19 @@ def log_submission_failure
     end
   end
 
-  def send_failure_mail
-    HCASubmissionFailureMailer.build(parsed_form['email'], google_analytics_client_id).deliver_now
+  def send_failure_email
+    email = parsed_form['email']
+    template_id = Settings.vanotify.services.health_apps_1010.template_id.form1010_ez_failure_email
+    api_key = Settings.vanotify.services.health_apps_1010.api_key
+
+    VANotify::EmailJob.perform_async(
```

Comment:
Hopefully using the job here will lower our chance of failures when sending the emails.